src/currency.js

```diff
@@ -1,14 +1,14 @@
 define([
 	"./core",
+	"./common/runtime-bind",
 	"./common/validate/cldr",
 	"./common/validate/default-locale",
 	"./common/validate/parameter-presence",
 	"./common/validate/parameter-type/currency",
 	"./common/validate/parameter-type/number",
 	"./common/validate/parameter-type/plain-object",
-	"./common/validate/plural-module-presence",
 	"./currency/code-properties",
-	"./currency/name-format",
+	"./currency/formatter/fn",
```

Comment:
`fn` is not a good module or function name. Its called as a function, so it much be a function, so what is it doing?